appserver/admingui/common/src/main/resources/org/glassfish/common/admingui/Strings.properties

```diff
@@ -389,8 +389,8 @@ log.multiLineMode=Multiline Mode:
 log.multiLineModeHelp=Start the log message body on a new line after the message header
 log.ConsoleLogFormat=Console Logging Format:
 log.ConsoleLogFormatHelp=Format for logging to the console
-log.DirectStdIO=Direct System Streams:
-log.DirectStdIOHelp=System.out and System.err are not redirected to logging, rather write to console directly.
+log.RedirectSystemStreams=Redirect System Streams:
```

Comment:
There is subtle issue with it. `RedirectSystemStreams` set to `true` does NOT redirect system streams. The default value of `false` causes the redirection.

Reply:
When true streams are redirected away from the server logs right?

Reply:
Yes, but not being redirected is their natural behavior. 

It's like a switch "Put driving wheel on opposite side". It's fine when you put the wheel on left-hand side when checked, put it will feel wrong for the vast majority of people knowing where driving wheels are supposed to be by default ;)

Reply:
@pdudits, to avoid the ambiguity, would it make much more sense to change it to something like "LogStandardStreams".

Reply:
Yes, that would work, as long as default value is `true`. Actually, even RedirectSystemStreams would work in such case (but I like this wording better). But this creates backward compatibility issue. When you migrate from older version, the behavior will change, as the older domain did not specify this property.

So I think you need to handle the situation where the property is not set, and treat it as `true` rather than usual default `false`.